config/application.yml.example

```diff
@@ -8,3 +8,5 @@ SAML_IDP_METADATA_FILE: /path/to/config/saml/metadata.xml
 SAML_CALLBACK_URL: http://localhost:3000/saml/auth/callback
 SAML_IDP_SSO_URL: https://example.com/idp/sso
 SAML_ISSUER: GIDS
+ARCHIVE_INSTITUTIONS: 'True'
+ACTIVE_RECORD_BATCH_SIZE: 1000
```

Comment:
This doesn't feel right to me, might we keep this batch size change in-line?

Reply:
adding gem 'config'